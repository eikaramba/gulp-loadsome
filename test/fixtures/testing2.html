<script src="http://loadso.me/jquery@1.11.1/angular@1.2.26/min.js"></script>