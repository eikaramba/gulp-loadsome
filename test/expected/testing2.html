<script src=".tmp/f99e8ea9ea84473abf6564879f094729.js"></script>